packages/legend-studio/src/components/editor/side-bar/WorkflowManager.tsx

```diff
@@ -256,8 +256,8 @@ const WorkflowJobLogsViewer = observer(
 const WorkflowExplorerContextMenu = observer(
   (
     props: {
-      workflowsState: WorkspaceWorkflowsState;
-      workflowState: WorkspaceWorkflowState;
+      workflowsState: WorkflowManagerState;
```

Comment:
--> `workflowManagerState`
Let's rename all `workflowsState` to `workflowMangerState`